### DISCUSSION TITLE
Octernships - Planned Support for Belize

### DISCUSSION THREAD

**Discussion by AlexDev404** [community/community#52934] [GitHub Education]
Hi everyone,

I was wondering if there are any plans to support Belize for Octernships? I'm curious because typically my country doesn't have access to opportunities like this, and I'm hoping to use Octernships for one of my future internships. I appreciate any information you can provide. 

Thanks for any help or clarification that can be provided.

**Comment by AlexDev404** [comment]
**Bump**

**Comment by AlexDev404** [comment]
Bumping this again.

**Comment by AlexDev404** [comment]
Two years later still no answer and it very much seems like Octernships is dead

**Reply by Wajahat73** [reply]
Don't know wat you talking bout

**Reply by AlexDev404** [reply]
> Don't know wat you talking bout

what? so why are you commenting then?
